src/components/simulation/SimulationButton.vue

```diff
@@ -2,22 +2,26 @@
   <div class="simulationButton">
     <v-menu :close-on-content-click="false" offset-y>
       <template #activator="{ on }">
-        <div class="split-btn">
+        <div class="btn-split text-no-wrap">
           <v-btn
             :disabled="state.disabled"
             :loading="state.project.simulation.state.running"
             @click="state.project.startSimulation()"
-            class="main-btn"
+            class="btn-main"
             outlined
+            title="Simulate"
           >
-            <v-icon left>mdi-play</v-icon>
-            <span
-              v-if="state.project.simulation.code.runSimulation"
-              v-text="'Simulate'"
-            />
-            <span v-else v-text="'Prepare'" />
+            <v-icon class="d-none d-lg-flex" left v-text="'mdi-play'" />
```

Comment:
I would like to keep that `center` - it was overwritten by your last commit.
```suggestion
            <v-icon class="d-none d-lg-flex" center v-text="'mdi-play'" />
```

Reply:
See next line. :-D

Reply:
Ah, I mixed it up and missed a `pull`. nvm^^